.github/workflows/build-test.yml

```diff
@@ -22,7 +22,8 @@ concurrency:
 env:
   NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
   NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID }}
-  NEXT_PUBLIC_CCTP_SUBGRAPH_BASE_URL: ${{ secrets.NEXT_PUBLIC_CCTP_SUBGRAPH_BASE_URL }}
```

Comment:
We should delete the same line on https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/.github/workflows/build-test.yml#L213C1-L214C1

Reply:
And L223 too